scss/_theme-overrides.scss

```diff
@@ -0,0 +1,515 @@
+:root {
```

Comment:
What is the purpose of this file?

Reply:
this is where we add our own theme for pelican, the colors, for example